WordPress/Classes/Services/BlogService.m

```diff
@@ -20,14 +20,30 @@
 #import "NSString+XMLExtensions.h"
 #import "TodayExtensionService.h"
 
+NSString *const LastUsedBlogURLDefaultsKey = @"LastUsedBlogURLDefaultsKey";
+NSString *const BlogEntity = @"Blog";
+NSString *const EditPostViewControllerLastUsedBlogURLOldKey = @"EditPostViewControllerLastUsedBlogURL";
+NSString *const BlogName = @"blogName";
+NSString *const IsVisiblePredicate = @"visible = YES";
+NSString *const IsWPComAndVisiblePredicate = @"account.isWpcom = YES AND visible = YES";
+NSString *const GetFeatures = @"wpcom.getFeatures";
+NSString *const VideopressEnabled = @"videopress_enabled";
+NSString *const XmlRpc = @"xmlrpc";
+NSString *const Standard = @"standard";
+NSString *const MinimumVersion = @"3.6";
+NSString *const DateCreatedGmt = @"date_created_gmt";
+NSString *const TimeZoneName = @"timezone";
+NSString *const GmtOffset = @"gmt_offset";
+NSString *const TimeZone = @"time_zone";
+NSString *const HttpsPrefix = @"https://";
+float const OneHourInSeconds = 60.0 * 60.0;
```

Comment:
Can we change this from `float` to `CGFloat`?  We've been trying to clean up C types as we come across them and use foundation types instead.